content/post/31-js-ipfs-0.26.md

```diff
@@ -0,0 +1,96 @@
+---
+date: 2017-09-13
+url: 30-js-ipfs-0-26
+title: js-ipfs 0.26.0 released
+author: David Dias
+---
+
+Today, we've released js-ipfs 0.26.0. This release brings bug fixes, perf improvements, git support, http gateway and more!
+
+TODO: replace by 0.26.0 launch tweet
+<blockquote class="twitter-tweet" data-conversation="none" data-lang="en"><p lang="en" dir="ltr">Good morning world 🌞<br><br>A fresh new js-ipfs is waiting for you:<br>+ ipfs@0.25.0<br>[08:52:21] Published to npm.<br><br>I bid you all a good day!</p>&mdash; David Dias (@daviddias) <a href="https://twitter.com/daviddias/status/885044398328819712">July 12, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+# 🔦 Highlights
+
+Here are some of the highlights for this new js-ipfs release. There were plenty more bug fixes, tiny perf improvements, doc improvements and others all across the js-ipfs module ecosystem. A really BIG THANK YOU to everyone that has been contributing with code, tests, examples and also bug reports! They help us identify situations that we miss with out tests.
+
+### New InterPlanetary Infrastructure
+
+You might have noticed some hiccups a couple of weeks ago, that was due to a revamp and improvement in our infrastructure that separated Bootstraper nodes from Gateway nodes. We've now fixed that by ensuring that a js-ipfs node connects to all of them. More nodes on https://github.com/ipfs/js-ipfs/issues/973 and https://github.com/ipfs/js-ipfs/pull/975. Thanks @lgierth for improving IPFS infra and for setting up all of those DNS websockets endpoints for js-ipfs to connect to :)
+
+### Now js-ipfs packs the IPFS Gateway as well
+
+You read it right! Now, js-ipfs packs the IPFS Gateway and launches it when you boot a daemon (`jsipfs daemon`). With this, you can use js-ipfs to access content in the browser just like you use to do in go-ipfs or use js-ipfs as a complete solution to add content in the network and preview it without leaving JS land, it is great for tooling. This was a [awesome contribution from @ya7ya and @harshjv](https://github.com/ipfs/js-ipfs/pull/968) that spent a lot of time adjusting and iterating on the implementation to make sure it would fit with the structure of js-ipfs, 👏🏽👏🏽👏🏽👏🏽.
```

Comment:
1. use period instead of comma before "it is great for tooling".
2. use "an awesome" instead of "a awesome" because next word (awesome) starts with a vowel
3. use "who spent" instead of "that spent" because it was people who spent a lot of time, etc.